common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -139,77 +182,177 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
 
 
   /////////////////////////////
```

Comment:
Thanks, thats a good steer. I have updated the comments to use this, and will use it in future.